src/overlays/actors/ovl_En_Tanron5/z_en_tanron5.c

```diff
@@ -55,90 +76,87 @@ static ColliderCylinderInit sCylinderInit = {
     { 70, 450, 0, { 0, 0, 0 } },
 };
 
-f32 D_80BE5DD0 = 1.0f;
+/**
+ * Multiplies the scale differently depending on whether the player is wearing the Giant's Mask or not.
+ * When the player is wearing the Giant's Mask, this value is smaller to make the player seem larger.
+ */
+static f32 sGiantModeScaleFactor = 1.0f;
 
-Vec2s D_80BE5DD4[] = {
-    { 0x4B0, 0x9C4 },  { -0x4B0, 0x9C4 },  { 0x4B0, -0x9C4 },  { -0x4B0, -0x9C4 }, { 0x9C4, 0x4B0 },
-    { -0x9C4, 0x4B0 }, { 0x9C4, -0x4B0 },  { -0x9C4, -0x4B0 }, { 0x3E8, 0x3E8 },   { -0x3E8, 0x3E8 },
-    { 0x3E8, -0x3E8 }, { -0x3E8, -0x3E8 }, { 0x000, -0x3E8 },  { 0x000, 0x3E8 },   { 0x3E8, 0x000 },
-    { -0x3E8, 0x000 }, { 0x000, -0x7D0 },  { 0x000, 0x7D0 },   { 0x7D0, 0x000 },   { -0x7D0, 0x000 },
+/**
+ * Stores the X and Z spawn positions for all of the ruins. Their Y spawn position is determined by the
+ * height of the floor, so there's no need to store it.
+ */
+static Vec2s sSpawnPositions[] = {
+    { 1200, 2500 },  { -1200, 2500 },  { 1200, -2500 },  { -1200, -2500 }, { 2500, 1200 },
+    { -2500, 1200 }, { 2500, -1200 },  { -2500, -1200 }, { 1000, 1000 },   { -1000, 1000 },
+    { 1000, -1000 }, { -1000, -1000 }, { 0, -1000 },     { 0, 1000 },      { 1000, 0 },
+    { -1000, 0 },    { 0, -2000 },     { 0, 2000 },      { 2000, 0 },      { -2000, 0 },
```

Comment:
```suggestion
    { 1200, 2500 },  // TWINMOLD_PROP_TYPE_RUIN_PILLAR_1 
    { -1200, 2500 },  // TWINMOLD_PROP_TYPE_RUIN_PILLAR_2
    { 1200, -2500 },  { -1200, -2500 }, { 2500, 1200 },
    { -2500, 1200 }, { 2500, -1200 },  { -2500, -1200 }, { 1000, 1000 },   { -1000, 1000 },
    { 1000, -1000 }, { -1000, -1000 }, { 0, -1000 },     { 0, 1000 },      { 1000, 0 },
    { -1000, 0 },    { 0, -2000 },     { 0, 2000 },      { 2000, 0 },      { -2000, 0 },
```
Could you add the types to these? and also `sDLists` and `sBaseScales`?

Reply:
Are you sure this is really a good idea? This would increase the number of lines that this data takes up from 11 to 60. Maybe for this position one, it might be worth it, but for the other two, I'm not sure it adds much. Every single value in `sBaseScales` is identical, and the type used for each entry in `sDLists` is fairly obvious.

Reply:
I personally still like it with the comments, no matter the num of lines it takes, since eventually these params will go into the scene spawn info, and it would make it much faster to search which pillar is mapped to which position/scale/dList, and if you were to add a pillar, it'd be easier to tell where to shift depending on where you add an enum for example. It just makes it easier to work with.

You could also just add a comment splitting it in 2 parts (the ruin pillars and the ruin pyramids)

Thoughts?

Reply:
Yeah, the modability argument is fair. It just felt like such a waste to expand `sBaseScales` any more than it needs to be, since it's basically useless. But I suppose if you wanted to actually *use* this, you would like to know which number corresponds to what ruin.

By the way, I *don't* think these params will ever move to the scene spawn info, because these are never actually spawned as part of the scene. Boss02 spawns the static instance of EnTanron5, which then spawns all the rest of them.